erigon-lib/diskutils/diskutils_darwin.go

```diff
@@ -3,14 +3,15 @@
 package diskutils
 
 import (
-	"fmt"
 	"syscall"
+
+	"github.com/ledgerwatch/log/v3"
 )
 
 func MountPointForDirPath(dirPath string) string {
 	var stat syscall.Statfs_t
 	if err := syscall.Statfs(dirPath, &stat); err != nil {
```

Comment:
psutil/v3 lib probably has cross-platform analog

Reply:
It doesn't, I am going to implement it and contribute to gopsutil